services/core-web/src/components/Forms/permits/conditions/ListItem.js

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from 'react';
+import PropTypes from "prop-types";
+import { Col, Row, Popconfirm, Button } from "antd";
+import { EDIT_OUTLINE_VIOLET, TRASHCAN } from "@/constants/assets";
+import AuthorizationWrapper from "@/components/common/wrappers/AuthorizationWrapper";
+import * as Permission from "@/constants/permissions";
+import ListItemForm from "@/components/Forms/permits/conditions/ListItemForm"
+
+const propTypes = {
+    condition: PropTypes.objectOf(PropTypes.any),
+    new: PropTypes.bool,
+    handleSubmit: PropTypes.func,
+    handleCancel: PropTypes.func,
+    initialValues: PropTypes.objectOf(PropTypes.any),
+};
+
+const defaultProps = {
+    condition: {
+        step: '',
```

Comment:
What is condition.step referring to?

Reply:
step is the dynamically generated number, letter, or roman numeral